src/cls/IPM/ResourceProcessor/SystemSetting.cls

```diff
@@ -0,0 +1,35 @@
+Class %IPM.ResourceProcessor.SystemSetting Extends %IPM.ResourceProcessor.Abstract
```

Comment:
Can a description be added to explain exactly what this does? Seems to be specific to some CSP settings? Feels like this should be deprecated in favor of the CPF resource processor (candidate for removal in v1.0.0).